<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ControlForm</class>
 <widget class="QWidget" name="ControlForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>631</width>
    <height>523</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_3">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="maximumSize">
          <size>
           <width>120</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>궁서</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>LIST</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="logoLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>350</width>
         <height>75</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTreeWidget" name="clientTreeWidget">
       <property name="maximumSize">
        <size>
         <width>350</width>
         <height>16777215</height>
        </size>
       </property>
       <column>
        <property name="text">
         <string>접속 상태</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>이름</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="font">
          <font>
           <family>궁서</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>SERVER</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTreeWidget" name="messageTreeWidget">
       <column>
        <property name="text">
         <string>IP</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>PORT</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>이름</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>메시지</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>시간</string>
        </property>
       </column>
      </widget>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="1">
        <widget class="QPushButton" name="controlBPushButton">
         <property name="styleSheet">
          <string notr="true">  font-family: Roboto;
  font-size: 15px;
  font-weight: bold;
  font-style: normal;
  line-height: 1.95;
  text-align: center;
  color: #fff;
background-color: #6073de;</string>
         </property>
         <property name="text">
          <string>Control B</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="controlCPushButton">
         <property name="styleSheet">
          <string notr="true">  font-family: Roboto;
  font-size: 15px;
  font-weight: bold;
  font-style: normal;
  line-height: 1.95;
  text-align: center;
  color: #fff;
background-color: #6073de;</string>
         </property>
         <property name="text">
          <string>Control C</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="controlDPushButton">
         <property name="styleSheet">
          <string notr="true">  font-family: Roboto;
  font-size: 15px;
  font-weight: bold;
  font-style: normal;
  line-height: 1.95;
  text-align: center;
  color: #fff;
background-color: #6073de;</string>
         </property>
         <property name="text">
          <string>Control D</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QPushButton" name="controlAPushButton">
         <property name="styleSheet">
          <string notr="true">  font-family: Roboto;
  font-size: 15px;
  font-weight: bold;
  font-style: normal;
  line-height: 1.95;
  text-align: center;
  color: #fff;
background-color: #6073de;</string>
         </property>
         <property name="text">
          <string>Control A</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
